packages/engine/src/scene/functions/loaders/PostprocessingFunctions.test.ts

```diff
@@ -1,5 +1,123 @@
+  let world: World
+  let entity: Entity
+  let postprocessingFunctions = proxyquire('./PostprocessingFunctions', {
+    '@xrengine/engine/src/common/functions/isClient': { isClient: true },
+    '../../../renderer/functions/configureEffectComposer': { configureEffectComposer: () => {} }
+  })
+
+  beforeEach(() => {
+    world = createWorld()
+    Engine.currentWorld = world
+    entity = createEntity()
+  })
+
+  const sceneComponentData = {
+    options: defaultPostProcessingSchema
+  }
+
+  const sceneComponent: ComponentJson = {
+    name: SCENE_COMPONENT_POSTPROCESSING,
+    props: sceneComponentData
+  }
+
+  describe('deserializePostprocessing()', () => {
+    it('does not create Postprocessing Component while not on client side', () => {
+      const _postprocessingFunctions = proxyquire('./PostprocessingFunctions', {
+        '@xrengine/engine/src/common/functions/isClient': { isClient: false }
+      })
+      _postprocessingFunctions.deserializePostprocessing(entity, sceneComponent)
+
+      const postprocessingComponent = getComponent(entity, PostprocessingComponent)
+      assert(!postprocessingComponent)
+    })
+
+    it('creates Postprocessing Component with provided component data', () => {
+      postprocessingFunctions.deserializePostprocessing(entity, sceneComponent)
+
+      const postprocessingComponent = getComponent(entity, PostprocessingComponent)
+      assert(postprocessingComponent)
+      assert.deepEqual(postprocessingComponent, sceneComponentData)
+    })
+
+    it('creates Postprocessing Object3D with provided component data', () => {
+      postprocessingFunctions.deserializePostprocessing(entity, sceneComponent)
+
+      assert(getComponent(entity, Object3DComponent)?.value, 'Postprocessing is not created')
+      assert(getComponent(entity, DisableTransformTagComponent))
+      assert(getComponent(entity, IgnoreRaycastTagComponent))
+    })
+
+    describe('Editor vs Location', () => {
+      it('creates Postprocessing in Location', () => {
+        addComponent(entity, EntityNodeComponent, { components: [] })
+
+        postprocessingFunctions.deserializePostprocessing(entity, sceneComponent)
+
+        const entityNodeComponent = getComponent(entity, EntityNodeComponent)
+        assert(!entityNodeComponent.components.includes(SCENE_COMPONENT_POSTPROCESSING))
+      })
+
+      it('creates Postprocessing in Editor', () => {
+        Engine.isEditor = true
+
+        addComponent(entity, EntityNodeComponent, { components: [] })
+
+        postprocessingFunctions.deserializePostprocessing(entity, sceneComponent)
+
+        const entityNodeComponent = getComponent(entity, EntityNodeComponent)
+        assert(entityNodeComponent.components.includes(SCENE_COMPONENT_POSTPROCESSING))
+
+        Engine.isEditor = false
+      })
+    })
+  })
+
+  describe('updatePostprocessing()', () => {
+    it('updates postprocessing stats', () => {
+      postprocessingFunctions.updatePostProcessing()
+      assert(true)
```

Comment:
missing assertion

Reply:
In `updatePostProcessing` function has only one line in which `configureEffectComposer` is called. So a successful return indicates passing of the test. Because any further tests are related to `configureEffectComposer` which will be in its test case collection.